ws-nextjs-app/pages/[service]/downloads/downloadsPageLayout.tsx

```diff
@@ -0,0 +1,86 @@
+/** @jsx jsx */
+
+import React, { useContext } from 'react';
+import { jsx } from '@emotion/react';
+import Heading from '#app/components/Heading';
+import Metadata from '#app/components/Metadata';
+import { ServiceContext } from '#app/contexts/ServiceContext';
+import TimeStampContainer from '#app/legacy/psammead/psammead-timestamp-container/src';
+import styles from './styles';
+import { PageProps } from './types';
+
+const downloadsPageLayout = ({ service, pageData }: PageProps) => {
```

Comment:
This will need to be `DownloadsPageLayout` (capital D) to make it a valid React component.